svc-cloud-validation/src/main/kotlin/gov/cdc/dex/transport/service/ValidationController.kt

```diff
@@ -0,0 +1,114 @@
+package gov.cdc.dex.transport.service
+
+import gov.cdc.dex.cloud.storage.CloudStorage
+import io.micronaut.http.HttpRequest
+import io.micronaut.http.HttpResponse
+import io.micronaut.http.MediaType
+import io.micronaut.http.annotation.Body
+import io.micronaut.http.annotation.Controller
+import io.micronaut.http.annotation.Delete
+import io.micronaut.http.annotation.Get
+import io.micronaut.http.annotation.PathVariable
+import io.micronaut.http.annotation.Post
+import io.micronaut.http.annotation.QueryValue
+import io.micronaut.http.multipart.CompletedFileUpload
+import org.slf4j.LoggerFactory
+import java.util.*
+//import gov.cdc.dex.hl7.Helper
+//import cdc.xlr.structurevalidator._
+import java.io.InputStreamReader
+import java.net.HttpURLConnection
+import java.net.URL
+import java.io.BufferedReader
+import java.io.OutputStream
+
+import com.google.gson.*
+
+
+/**
+ *
+ *
+ * @Created - 6/23/23
+ * @Author USY6@cdc.gov
+ */
+@Controller("/validation")
+class ValidationController(private val cloudStorage: CloudStorage) {
+    private val log = LoggerFactory.getLogger(ValidationController::class.java.name)
+    val gson: Gson = GsonBuilder().serializeNulls().create()
+    val redactorUrl = System.getenv("REDACTOR_URL") + "/api/redactorReport"
+    val structureUrl = System.getenv("STRUCTURE_URL") + "/api/structure"
+    val validationUrl = System.getenv("MMG_VALIDATOR_URL") + "/api/validate-mmg"
+    val elrType = System.getenv("ELR_TYPE")
```

Comment:
Is it possible to extract these constant values? Instead of using multiple `System.getenv` calls, extract the constant values into named constants at the class level. This improves code readability and makes it easier to modify the values in one place.

```
 companion object {
        const val REDACTOR_REPORT_URL = "${System.getenv("REDACTOR_URL")}/api/redactorReport"
        const val STRUCTURE_URL = "${System.getenv("STRUCTURE_URL")}/api/structure"
        const val VALIDATION_URL = "${System.getenv("MMG_VALIDATOR_URL")}/api/validate-mmg"
        const val ELR_TYPE = System.getenv("ELR_TYPE")
    }
 ```

Reply:
After discussing this change with team members in a shared teams session it was decided to keep this part as is.